docs/Using-Fleet/configuration-files/README.md

```diff
@@ -221,11 +224,6 @@ spec:
     host_expiry_enabled: true
     host_expiry_window: 10
   features:
-    # "additional" information to collect from hosts along with the host
```

Comment:
This was removed from the YAML and moved to the corresponding setting's section below, for consistency.